runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -4087,6 +4101,10 @@ TR::Register *J9::Power::TreeEvaluator::VMinstanceOfEvaluator(TR::Node *node, TR
    if (objectClassReg)
       srm->reclaimScratchRegister(objectClassReg);
 
+   if (generateBranchToOOL) {
+      if (comp->getOption(TR_TraceCG)) traceMsg(comp, "%s: Emitting branch to OOL in case of single sequence of Helper\n",node->getOpCode().getName());
+      generateLabelInstruction(cg, TR::InstOpCode::b, node, nextSequenceLabel);
+   }
    if (true)
```

Comment:
also, a general comment:  separate label instructions for dependency and helperReturn might be incorrect. imagine there are spills generated at the dependency label and they are not executed in runtime when the control returns from the helper call.